.github/CODEOWNERS

```diff
@@ -0,0 +1,3 @@
+# Global owner for everything in this repo
+
+- @rt4914
```

Comment:
This one is still incorrect.

It should 
`* @rt4914` currently its missing the * at the start.